clusters/app.ci/gh-pull-through-cache/gh-pull-through-cache.yaml

```diff
@@ -0,0 +1,112 @@
+apiVersion: v1
+kind: List
+items:
+- apiVersion: v1
+  kind: ConfigMap
+  metadata:
+    name: gh-pull-through-cache
+    namespace: jguzik-playground
```

Comment:
@jupierce target namespace would be ci right?

Reply:
Anything should work, but that's where I'd put it.